src/script/bitcoinconsensus.cpp

```diff
@@ -69,7 +69,7 @@ struct ECCryptoClosure
 ECCryptoClosure instance_of_eccryptoclosure;
 }
 
-int bitcoinconsensus_verify_script(const unsigned char *scriptPubKey, unsigned int scriptPubKeyLen, uint64_t amount,
+int bitcoinconsensus__verify_script(const unsigned char *scriptPubKey, unsigned int scriptPubKeyLen, CAmount amount,
```

Comment:
Seems it is. Perhaps make it static, and give it a more obvious name ("_internal" ?)